docs/android-setup.md

```diff
@@ -0,0 +1,90 @@
+# Installation
+
+In some cases, simply doing `react-native link` will correctly set up the tracker on Android.
+However, if you encounter any issues (either with the command itself or at compile-time/run-time),
+you should revert any changes made by `react-native link` and try the following steps, instead.
+
+Add the following to your `settings.gradle`:
+
+  ```groovy
+  include ':react-native-heap'
+  project(':react-native-heap').projectDir =
+      new File(rootProject.projectDir, '../node_modules/@heap/react-native-heap/android')
+  ```
+
+Then add the following to `app/build.gradle`:
+
+  ```groovy
+  implementation project(':react-native-heap')
+  ```
+
+Import the package and add it to your app's MainApplication:
+
+  ```java
+  @Override
+  import com.heapanalytics.reactnative.RNHeapLibraryPackage;
+  ...
+  protected List<ReactPackage> getPackages() {
+    return Arrays.<ReactPackage>asList(
+      new MainReactPackage(),
+      new RNHeapLibraryPackage()
+    );
+  }
+  ```
+
+If you are seeing runtime warnings at startup mentioning `Heap: Could not find BuildConfig`, add the
+following line to the resources section of `res/values/strings.xml`:
+
+  ```xml
+  <string name="com.heapanalytics.android.buildConfigPkgName">com.your_package_name</string>
+  ```
+
+and replace `com.your_package_name` with the package name from the manifest tag in `AndroidManifest.xml`.
+
+# Initialization
+
+- This library needs to be initialized so events are sent to the right application/environment ID. 
+Place a `heap.config.json` file at the root of your application's repository with this structure, substituting your own application/environment IDs.
+
+  ```json
+  {
+    "default": {
+      "heapAutoInit": true
+    },
+    "dev": {
+      "heapAutoInit": true,
+      "heapAppId": "11"
+    },
+    "prod": {
+      "heapAppId": "12",
+      "heapAutoInit": true
+    }
+  }
+  ```
+
+- Note that different application/environment IDs can be set for development and production. Heap can also be enabled/disabled on a per-environment basis. Values in `default` are used if a key is missing in either `dev` or `prod`.
```

Comment:
What exactly triggers the use of dev or prod configurations?

Reply:
We use the `__DEV__` variable, which RN sets for non-release builds.

https://facebook.github.io/react-native/docs/javascript-environment#polyfills
https://facebook.github.io/react-native/docs/performance#profiling

Reply:
I probably should have clarified that I was asking the question primarily because I think we should include that information in the docs

Reply:
Gotcha 👍